cluster/src/main/java/org/apache/iotdb/cluster/log/manage/serializable/SyncLogDequeSerializer.java

```diff
@@ -71,6 +73,8 @@
   private long maxAvailableTime = Long.MAX_VALUE;
```

Comment:
I think you should also change the names and comments of `minAvailableTime` and `maxAvailableTime`.